pkg/nsx/services/securitypolicy/expand.go

```diff
@@ -20,7 +20,7 @@ import (
 
 // When a rule contains named port, we should consider whether the rule should be expanded to
 // multiple rules if the port name maps to conflicted port numbers.
-func (service *SecurityPolicyService) expandRule(obj *v1alpha1.SecurityPolicy, rule *v1alpha1.SecurityPolicyRule,
+func (service *Service) expandRule(obj *v1alpha1.SecurityPolicy, rule *v1alpha1.SecurityPolicyRule,
```

Comment:
service *Service should be from firewall.go, but this naming is making reader confused with the Service defined in common.go 

In  firewall.go:
Service struct {
	common.Service
	Client              client.Client
	Client              client.Client
	NSXClient           *nsx.Client
	NSXConfig           *config.NSXOperatorConfig
	GroupStore          cache.Indexer
	GroupStore          cache.Indexer
	SecurityPolicyStore cache.Indexer
	SecurityPolicyStore cache.Indexer
	RuleStore           cache.Indexer
	RuleStore           cache.Indexer
}

Also considering code maintenance, I suggest that we still use SecurityPolicyService rather than Service.

Reply:
ok